src/averageAges.js

```diff
@@ -52,7 +58,15 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  return +people
+    .filter(child =>
+      people.some(mother => onlyWithSon ? child.mother === mother.name
```

Comment:
```suggestion
      people.some(mother => onlyWithSon
        ? child.mother === mother.name
```
Краще перенести на новий рядок